macros/CanvasSidebar.ejs

```diff
@@ -102,54 +102,69 @@ var text = mdn.localStringMap({
 <section id="Quick_Links">
  <ol>
   <li><a href="/<%=locale%>/docs/Web/API/Canvas_API"><strong><%=text['Canvas_API']%></strong></a></li>
-  <li data-default-state="open"><a href="/<%=locale%>/docs/Web/API/Canvas_API/Tutorial"><strong><%=text['Canvas_tutorial']%></strong></a>
-   <ol>
-    <li><a href="/<%=locale%>/docs/Web/API/Canvas_API/Tutorial/Basic_usage"><%=text['Basic_usage']%></a></li>
-    <li><a href="/<%=locale%>/docs/Web/API/Canvas_API/Tutorial/Drawing_shapes"><%=text['Drawing_shapes']%></a></li>
-    <li><a href="/<%=locale%>/docs/Web/API/Canvas_API/Tutorial/Applying_styles_and_colors"><%=text['Applying']%></a></li>
-    <li><a href="/<%=locale%>/docs/Web/API/Canvas_API/Tutorial/Drawing_text"><%=text['Text']%></a></li>
-    <li><a href="/<%=locale%>/docs/Web/API/Canvas_API/Tutorial/Using_images"><%=text['Using_images']%></a></li>
-    <li><a href="/<%=locale%>/docs/Web/API/Canvas_API/Tutorial/Transformations"><%=text['Transformations']%></a></li>
-    <li><a href="/<%=locale%>/docs/Web/API/Canvas_API/Tutorial/Compositing"><%=text['Compositing']%></a></li>
-    <li><a href="/<%=locale%>/docs/Web/API/Canvas_API/Tutorial/Basic_animations"><%=text['Basic_animations']%></a></li>
-    <li><a href="/<%=locale%>/docs/Web/API/Canvas_API/Tutorial/Advanced_animations"><%=text['Advanced_animations']%></a></li>
-    <li><a href="/<%=locale%>/docs/Web/API/Canvas_API/Tutorial/Pixel_manipulation_with_canvas"><%=text['Pixel_manipulation']%></a></li>
-    <li><a href="/<%=locale%>/docs/Web/API/Canvas_API/Tutorial/Hit_regions_and_accessibility"><%=text['Hitregions']%></a></li>
-    <li><a href="/<%=locale%>/docs/Web/API/Canvas_API/Tutorial/Optimizing_canvas"><%=text['Optimizing']%></a></li>
-    <li><a href="/<%=locale%>/docs/Web/API/Canvas_API/Tutorial/Finale"><%=text['Finale']%></a></li>
-   </ol>
+  <li class="toggle">
```

Comment:
Here's the ``{{CanvasSidebar}}`` on https://developer.mozilla.org/en-US/docs/Web/API/Canvas_API/Tutorial/Drawing_shapes. It looks good 👍. The section headers are no longer bold, and the spacing is a little different. The scraper always enables the black Jump Bar, but it is off on the production page.

<img width="719" alt="pr 789 canvas sidebar" src="https://user-images.githubusercontent.com/286017/45326953-238d3000-b51b-11e8-9028-a3f5f53f59c1.png">